unit_tests/onthefly_interpolation/problem.c

```diff
@@ -51,7 +51,7 @@ int main(int argc, char* argv[]){
     
     // Compare results
     for (int i=0; i<Noutput; i++){
-        assert(fabs(x_direct[i]-x_interpolate[i]) < 1e-13); // Check that results agree
+        assert(fabs(x_direct[i]-x_interpolate[i]) < 2e-13); // Check that results agree
```

Comment:
I believe the switch to SPICE is making the interpolate function agree ever so slightly less. After doing many comparisons with JPL Horizons and seeing equal or better residuals (compared to the `.440` version of ASSIST), I'm inclined to believe this has to do more with interpolation than it does less accuracy with the integration.